inputs.nolong.yml.example

```diff
@@ -0,0 +1,288 @@
+# This file specifies the attributes for the configurable controls
+# used in the RHEL 7 DISA STIG.
+
+# Controls that are known to consistently have long run times can be disabled with this attribute
+disable_slow_controls: true
+
+# V-72081 - 'monitor_kernel_log', (bool)
+# Set this to false if your system availability concern is not documented or
+# there is no monitoring of the kernel log
+monitor_kernel_log: true
+
+# V-71849
+# list of system files that should be allowed to change from an rpm verify point of view
+rpm_verify_perms_except: []
+
+# V-71855
+# list of system files that should be allowed to change from an rpm verify point of view
+rpm_verify_integrity_except: []
+
+# V-71859 (already set to true)
+banner_message_enabled: 'true'
+
+# V-72211 (default: false)
+log_aggregation_server: false
+
+# V-72047 (default: [])
+# Known application groups that are allowed to have world-writeable files or directories
+application_groups: []
+
+# V-72307
+x11_enabled: false
+
+# Accounts that are not allowed on the system (Array)
+disallowed_accounts: [
+  'games',
+  'gopher',
+  'ftp',
+]
+
+# Accounts of known managed users (Array)
+user_accounts: []
+
+# System accounts that support approved system activities. (Array)
+known_system_accounts: [
+    'root',
+    'bin',
+    'daemon',
+    'adm',
+    'lp',
+    'sync',
+    'shutdown',
+    'halt',
+    'mail',
+    'operator',
+    'nobody',
+    'systemd-bus-proxy',
+    'systemd-network',
+    'dbus',
+    'polkitd',
+    'tss', #  Account used by the trousers package to sandbox the tcsd daemon
```

Comment:
The accounts from this point down should not be present on a default system unless required by installed packages.

Honestly, this whole control is pretty much nonsense. But, if we're going to go with the letter of the check, we should knock out the default accounts.

`sshd` can probably stay for obvious reasons, but it technically also isn't necessary.

Reply:
Okay, I removed the extra accounts from the various input files.